src/Statement.py

```diff
@@ -46,8 +56,11 @@ def is_fee(self, booking_type):
 
     def get_date(self):
         """ get the date of the statement """
+        date_to_parse = self._statement[self._config.get_booking_date()]
```

Comment:
Why is this required? Are there statements without date? not so nice. hmm.

Reply:
I had a downloaded csv with a line without date, but in the file I downloaded today all entries have a date.

So we could keep it to be more resilient for missing dates, as otherwise the parser just throws an error and stops.